tests/version-number.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(non_fmt_panic)]
+#![allow(non_fmt_panics)]
```

Comment:
if the old version was a miss-spelling and thus presumably inactive, then do we need this?